app/helpers/checkout_helper.rb

```diff
@@ -19,7 +19,7 @@ def checkout_adjustments_for(order, opts = {})
     adjustments.reject! { |a| a.originator_type == 'EnterpriseFee' && a.source_type != 'Spree::LineItem' }
     unless exclude.include? :admin_and_handling
       adjustments << Spree::Adjustment.new(
-        label: I18n.t(:orders_form_admin), amount: enterprise_fee_adjustments.map(&:amount).sum
+        label: I18n.t(:orders_form_admin), amount: enterprise_fee_adjustments.sum(:amount)
```

Comment:
I think that your assumption of having an AR relation isn't correct here. `adjustments` are an array and `enterprise_fee_adjustments` are an array, too. You can simplify this though:
```suggestion
        label: I18n.t(:orders_form_admin), amount: enterprise_fee_adjustments.sum(&:amount)
```